src/endpoint/s3/ops/s3_post_bucket_delete.js

```diff
@@ -22,6 +23,11 @@ async function post_bucket_delete(req) {
         objects
     });
 
+    if (objects.length > 1000) {
+        dbg.error('cannot delete more than 1000 objects');
+        throw new S3Error(S3Error.MalformedACLError);
```

Comment:
Why did you choose that error? (please share the relevant part in the documentation).

Reply:
aws documentation doesn't specify the error returned. only that you can't delete more than 1000 objects: https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html 
this is the error returned by aws when sending such command